core/templates/dev/head/pages/exploration_player/RefresherExplorationConfirmationModalService.js

```diff
@@ -35,7 +35,21 @@ oppia.factory('RefresherExplorationConfirmationModalService', [
             function($scope, $uibModalInstance, $window) {
               $scope.confirmRedirect = function() {
                 redirectConfirmationCallback();
-                var url = '/explore/' + refresherExplorationId;
+                var collectionId = UrlService.getUrlParams().collection_id;
+                var parentIdList = UrlService.getQueryFieldValuesAsList(
+                  'parent');
+                var urlTemplate = '/explore/<refresher_exploration_id>';
+                var interpolationValues = {};
+                interpolationValues.refresher_exploration_id =
+                  refresherExplorationId;
+                url = UrlInterpolationService.interpolateUrl(
```

Comment:
Missing var

Also replace this and the above few lines with

```
    var EXPLORATION_URL_TEMPLATE = '/explore/<exploration_id>'
    url = UrlInterpolationService.interpolateUrl(
      EXPLORATION_URL_TEMPLATE, {
        exploration_id: refresherExplorationId
      }));
```

Reply:
Done